### DISCUSSION TITLE
Bienvenidos a las discusiones en data-mining-2022!

### DISCUSSION THREAD

**Discussion by furrutiav** [furrutiav/data-mining-2022#8] [Announcements]
## 👋 ¡Bienvenido!
  Usamos los Debates como un lugar para conectar con otros miembros de nuestra comunidad. Esperamos que:
  * Haga preguntas que se pregunte.
  * Comparta ideas.
  * Se relacione con otros miembros de la comunidad.
  * Dé la bienvenida a los demás y tenga una mentalidad abierta. Recuerda que esta es una comunidad que
  construimos juntos 💪.

  Para empezar, comenta abajo con una presentación tuya y cuéntanos qué haces con esta comunidad.

**Comment by camilocarvajalreyes** [comment]
ola
